reversestring-JALY/prompts.md

```
@@ -0,0 +1,5 @@
+Chat GPT - Thinking
+
+Act as JavaScript expert. Given this index.html file, where you can see that the script.js file is referenced there, and given the attached image. Do as follows: * Provide me an edited version for "index.html" file * A CSS file if required, this one should be reference by the index.html file * The "script.js" file The web application expected behavior is as follows: * There must be a fixed title "Reverse string" H1 type * There must be a text field for the users to introduce a text. Max 200 characters * There must be a blue "Reverse" button with the icon, looking exactly like the one on the attached image by using CSS. This button should be visible only when the previous text field has at least 3 characters; otherwise, it should be hidden. When this button is clicked, the script.js file should reverse the order of the characters introduced by the user on the text field. This should be shown below on a black label field * There must be a grey "Copy" button with the icon, looking exactly like the one on the attached image by using CSS. When this button is clicked, the label text should be copied to the user's clipboard Do you have any question for me before generating the 3 requested files?
```

markdownlint-cli2 (0.21.0)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)